ee/server/requestSeatsRoute.ts

```diff
@@ -0,0 +1,20 @@
+import { IncomingMessage, ServerResponse } from 'http';
+
+import { Meteor } from 'meteor/meteor';
+import { WebApp } from 'meteor/webapp';
+
+import { getSeatsRequestLink } from '../app/license/server/getSeatsRequestLink';
+import { Analytics } from '../../server/sdk';
+import { isEnterprise } from '../app/license/server/license';
```

Comment:
Hey @g-thome , following the changes that Rodrigo suggested to #23256 , I think we should create a regression to fix this import too :grimacing:

Reply:
I believe Rodrigo sees that import as problematic because I was importing EE outside of EE, so I think doing it over here is ok. Am I right? @rodrigok